indico/modules/events/abstracts/models/comments.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core.db import db
+from indico.core.db.sqlalchemy import UTCDateTime
+from indico.util.date_time import now_utc
+from indico.util.string import format_repr, return_ascii, text_to_repr
+
+
+class AbstractComment(db.Model):
```

Comment:
missing relationship to `Abstract` (and if we keep soft-deletion for comments set `primaryjoin` in the backref to skip deleted ones, also sort the relationship by `created_dt`)